### ISSUE TITLE
VSCode SonarLint does not report XREF-related issues

### ISSUE THREAD

**Issue by maelstraggiotti** [Riverside-Software/sonar-openedge#1180]
Hi,

I struggle to make the XREF-related issues to work in VSCode. 
The Xref file is correctly generated on compile, but the SonarLint verbose output shows an incorrect/incomplete path to search for.

```
Source: 
<workspace_root>\product\server\work\cabl_test.p

Rcode:
<workspace_root>\product\run\server\work\cabl_test.r

Xref:
<workspace_root>\product\run\server\.pct\work\cabl_test.p.xref

Sonarlint output:
[Debug - 11:09:27.565] XREF file: <workspace_root>\work\cabl_test.xref.xml - Found: false
[Debug - 11:09:27.565] Empty XREF file
```

OpenEdge 12.8.6
tested with:
openedge-abl-lsp-1.18.2 / openedge-abl-lsp-1.19.6
sonarlint-abl-4.11.99001 / sonarlint-abl-4.15.99001

Thanks

**Comment by gquerret** [comment]
I confirm the XREF file is not read correctly from VS Code Sonarlint. That will be fixed in next version.

**Comment by maelstraggiotti** [comment]
Hi @gquerret, got any ETA for that fix ?
Thanks

**Comment by gquerret** [comment]
Didn't see your previous message. That will be part of the next sonar-openedge (3.2) release, planned for end of July.
